projects/nlp/hello_classification_model.py

```diff
@@ -23,44 +23,55 @@
 helpStr = """
 Simple script that explains how to run classification models.
 
-Example invocations:
+The standard dataset is 10 documents of three-word sentences, each classified
+into one of two categories. Bag of words (BOW) models should be able to
+discriminate them.
+Test data will be a copy of training data plus two additional documents. The
+first is an incorrectly labeled version of a training sample. The second is
+semantically similar to one of the training samples.
+Expected classification accuracy using CIO encoder is 9 out of 10 = 90%
+Expected classification accuracy using keywords encoder is 8 out of 10 = 80%
 
+Example invocations:
 python hello_classification_model.py -m keywords
-python hello_classification_model.py -m docfp
-python hello_classification_model.py -c data/network_configs/sensor_knn.json -m htm -v 2
-python hello_classification_model.py -c data/network_configs/tp_knn.json -m htm
-
+python hello_classification_model.py -m docfp -v 0
+python hello_classification_model.py -m htm \
+  -c data/network_configs/sensor_simple_tp_knn.json \
+  --retina en_associative_64_univ
 """
 
 import argparse
 import copy
+from prettytable import PrettyTable
+from textwrap import TextWrapper
+from tqdm import tqdm
 
 from htmresearch.frameworks.nlp.classification_model import ClassificationModel
 from htmresearch.frameworks.nlp.classify_htm import ClassificationModelHTM
 from htmresearch.frameworks.nlp.model_factory import (
   createModel, getNetworkConfig)
+from htmresearch.support.csv_helper import mapLabelRefs, readCSV
+
+
+wrapper = TextWrapper(width=80)
+
+
+
+def getData(dataPath):
```

Comment:
Because the same "hello classification" data is read in with the simple_labels.py script I did the same here as not to duplicate the data. Sure, I can switch it, we want people to easily grasp the classification model API with this example, not confuse them.